.github/workflows/test.yaml

```
@@ -24,16 +25,43 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+          cache: "pip"
```

from https://github.com/rungalileo/rungalileo/blob/main/.github/workflows/test.yaml#LL48C11-L48C50

```suggestion
          cache: "pip"
          cache-dependency-path: "pyproject.toml"
```